sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -633,39 +686,15 @@ export class CommandStem<A extends Args[] | { [argName: string]: any } = [] | {}
       if (Array.isArray(value)) {
         // @ts-ignore : 'RepeatArgument' found in JSON Spec
         let repeatArg: RepeatArgument = {
-          value: value
-            .map(arg => {
-              return this.convertArgsToInterfaces(arg);
-            })
-            .flat(),
+          value: value.map(arg => {
+            return this.convertRepeatArgs(arg);
```

Comment:
What if the repeating arguments have nested repeating arguments? We probably want to have some recursion for that.